frontend/packages/kubevirt-plugin/integration-tests/tests/utils/types.ts

```diff
@@ -1,20 +1,32 @@
+import { DISK_SOURCE } from './consts';
+
 export type ProvisionOption = {
   method: string;
   source?: string;
 };
 
 export type NetworkResource = {
   name: string;
+  model: string;
   mac: string;
-  binding: string;
-  networkDefinition: string;
+  network: string;
+  type: string;
+};
+
+export type DiskSourceConfig = {
+  source: DISK_SOURCE;
+  PVCName?: string;
+  PVCNamespace?: string;
+  URL?: string;
+  container?: string;
 };
 
 export type StorageResource = {
```

Comment:
I think it makes more sense to put the source outside of the sourceConfig - this way we can use the same approach for BLANK as well

Reply:
updated